static/media/offline.svg

```diff
@@ -0,0 +1,19 @@
+    <desc>Created with Sketch.</desc>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Artboard" transform="translate(-58.000000, -52.000000)">
+            <g id="Group-2" transform="translate(59.000000, 53.000000)">
+                <g id="wifi-signal-(1)" transform="translate(15.931542, 13.992074) rotate(-45.000000) translate(-15.931542, -13.992074) translate(6.431542, 4.492074)" fill="#f27041" fill-rule="nonzero">
+                    <path d="M2.12621086,18.143666 C3.29881062,18.143666 4.28785857,17.1900549 4.28785857,16.0174551 C4.28785857,14.8448554 3.29881062,13.8558074 2.12621086,13.8558074 C0.953611099,13.8558074 0,14.8448554 0,16.0174551 C0,17.1900549 0.953611099,18.143666 2.12621086,18.143666 Z" id="Path"></path>
+                    <path d="M0.531552715,10.6310543 L1.59465814,10.6310543 C4.82384089,10.6310543 7.5126117,13.3196867 7.5126117,16.5490079 L7.5126117,17.6121133 C7.5126117,17.9057131 7.75056461,18.143666 8.04416442,18.143666 L9.10726985,18.143666 C9.40086966,18.143666 9.63882256,17.9057131 9.63882256,17.6121133 L9.63882256,16.5490079 C9.63882256,12.1454258 5.99824017,8.50484344 1.59465814,8.50484344 L0.531552715,8.50484344 C0.237952904,8.50484344 0,8.74279634 0,9.03639615 L0,10.0995016 C0,10.3931014 0.237952904,10.6310543 0.531552715,10.6310543 Z" id="Path"></path>
+                    <path d="M0.531552715,6.37863258 L1.59465814,6.37863258 C7.17250103,6.37863258 11.7650334,10.971165 11.7650334,16.5490079 L11.7650334,17.6121133 C11.7650334,17.9057131 12.0029863,18.143666 12.2965861,18.143666 L13.3596916,18.143666 C13.6532914,18.143666 13.8912443,17.9057131 13.8912443,17.6121133 L13.8912443,16.5490079 C13.8912443,9.79690414 8.34676186,4.25242172 1.59465814,4.25242172 L0.531552715,4.25242172 C0.237952904,4.25242172 0,4.49037462 0,4.78397443 L0,5.84707986 C0,6.14067967 0.237952904,6.37863258 0.531552715,6.37863258 Z" id="Path"></path>
+                    <path d="M1.59465814,0 L0.531552715,0 C0.237952904,0 0,0.237952904 0,0.531552715 L0,1.59465814 C0,1.88825796 0.237952904,2.12621086 0.531552715,2.12621086 L1.59465814,2.12621086 C9.52102272,2.12621086 16.0174551,8.62264328 16.0174551,16.5490079 L16.0174551,17.6121133 C16.0174551,17.9057131 16.255408,18.143666 16.5490079,18.143666 L17.6121133,18.143666 C17.9057131,18.143666 18.143666,17.9057131 18.143666,17.6121133 L18.143666,16.5490079 C18.143666,7.44838242 10.6952836,0 1.59465814,0 Z" id="Path"></path>
+                </g>
+                <path d="M15.4049994,31.5599989 C23.9129457,31.5599989 30.8099989,24.6629457 30.8099989,16.1549994 C30.8099989,7.64705318 23.9129457,0.75 15.4049994,0.75 C6.89705318,0.75 -2.13162821e-14,7.64705318 -2.13162821e-14,16.1549994 C-2.13162821e-14,24.6629457 6.89705318,31.5599989 15.4049994,31.5599989 Z M26.373692,6.49794837 L3.94869287,25.8029484" id="Combined-Shape" stroke="#f27041" stroke-width="1.94999993"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
We would ideally introduce a new "offline" icon as part of [our glyph set](https://v-next--cloudfour-patterns.netlify.app/?path=/story/design-icons--page) instead of adding an icon to the `static` directory.

This should match the weight, appearance, etc. of our other icons and be included in the Illustrator file in our shared Drive for consistency.